railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -55,3 +55,6 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 <% end -%>
+
+# Uncomment this, to increase the performance
```

Comment:
IMO this comment needs to be more descriptive as to why / how performance will be improved - especially because in some environments (Windows, for instance) the "fast" version requires an additional gem.